asv_bench/benchmarks/groupby.py

```diff
@@ -464,7 +465,12 @@ def setup(self, dtype, method, application, ncols):
             # DataFrameGroupBy doesn't have these methods
             raise NotImplementedError
 
-        ngroups = 1000
+        if method == "describe" and ncols == 5:
```

Comment:
could do this without the ncols check?

Reply:
im imagining looking at results and seeing the ncols==5 case being faster than the ncols==1 case (or just much less than 5x slower) and being confused until i remember this special case

Reply:
i agree or add a comment here

Reply:
Yes, certainly (I initially used `ngroups = 100` for each of the three methods for both ncols parameters, but then only describe for ncols=5 was still on the slow side. 
Using the smaller ngroups for both cases of describe will make the ncols=1 still faster, but that can't hurt I suppose.